src/main/java/org/spongepowered/common/mixin/core/data/MixinDataHolder.java

```diff
@@ -413,9 +413,4 @@ public DataContainer toContainer() {
         return Collections.emptyList();
     }
 
-    @Override
-    public DataHolder copy() {
```

Comment:
I don't see your changes in a recent commit @Mumfrey I'd appreciate if someone could narrow it down for me.